downstream/modules/platform/proc-deploy-eda-controller-with-aap-operator-ocp.adoc

```diff
@@ -0,0 +1,45 @@
+
+[id="proc-deploy-eda-controller-with-aap-operator-ocp_{context}"]
+
+
+
+.Prerequisites
+
+* You have installed {OperatorPlatform} on {OCPShort}.
+* You have installed and configured {ControllerName}.
+
+.Procedure
+
+. Navigate to menu:Operators[Installed Operators].
+
```

Comment:
I was testing this out in my test instance of AAP on OpenShift and I think there's more to this procedure:
1. Select Operators->Installed Operators. 
2. Locate and select your installation of Ansible Automation Platform.
3. Locate the EDA modal under Provided APIs and click Create instance.
This takes you to the Form View to customize your installation.

Reply:
@dcdacosta  - for the first step, do we still use the accessibility method for this: "Select menu:Operators[installed Operators]? I'll do it that way for now and you can let me know if it's correct when you peer review this second attempt.

Reply:
Also, @dcdacosta, should the step "Create a new Event-Driven Ansible custom resource" be removed now that you've added the first 3 steps above?

Reply:
@jself-sudoku, I think you can delete it since that is what you're doing in this procedure. If you wanted to keep it, I would add it to the intro of the procedure as "Use this procedure to create a new Event-Driven Ansible custom resource".